charts/kapacitor/TICKscripts/README.md

```diff
@@ -24,3 +24,80 @@ Once the scripts have been copied to the pod, we can now deploy them as Kapacito
+		.slack()
+		.stateChangesOnly()
+```
+
+There are a few things to note about this script:
+ 1. In the warn function is where the logic goes that will trigger the alert to send the error/alert message
+ 2. The alert message is specified in the `.message()` function.
+ 3. Calling `.slack()` tells Kapacitor to send the message to the Slack webhook that Kapacitor was set up with.
+ 4. The window clause is setting the script to only look at the last minute of data, and to only send one alert per minute.
+ This stops a spam of alerts if something enters an alert state for an extended period of time. 
+ 5. The double brackets in the message is the syntax for embedding measurement information into the error message,
+ in this case, the name of the host computer that is experiencing system blocking. 
+ 6. The `.stateChangesOnly()` function also reduces alert spamming by only sending an alert if the system goes from an "ok" state to an "alert" state.
+ meaning if something fails continuously for 40 minutes, instead of getting 40 alerts (one for every minute), only the first alert will be sent.
+
+ ### Batch Scripts
+ Over the course of this project, the team only ran into one instance where a batch script was required to achieve the desired functionality.
+ This was the container restarts alert. This was because in order to identify how many times a container had restarted in the last hour,
+ we had to find the difference between the current restarts count and the count from an hour prior.
+ Meaning that a query for current data needed to be joined with a query of data from the same measurement that was an hour old.
+Here is the script:
+```
+dbrp "telegraf"."autogen"
+
+var cur_data = batch
+	|query('select container,max(counter) from "telegraf"."autogen"."kube_pod_container_status_restarts_total"')
+	.groupBy('container')
+	.period(1m)
+	.every(1m)
+
+var prev_data = batch	
+	|query('select container,max(counter) from "telegraf"."autogen"."kube_pod_container_status_restarts_total"')
+	.groupBy('container')
+	.period(1m)
+	.every(1m)
+	.offset(1h)
+	|shift(1h)
+
+cur_data
+	|join(prev_data)
+		.as('cur','prev')
+		.tolerance(1m)
+	|alert()
+		.warn(lambda: (("cur.max" - "prev.max") >=5 AND ("cur.max" - "prev.max") <= 10))
+		.crit(lambda: (("cur.max" - "prev.max") >10))
+		.message('warning: Node {{ index .Tags "container" }} has gone from {{ index .Fields "prev.max" }} restarts to {{ index .Fields "cur.max" }} restarts in the past hour')
+		.slack()
+```
+Some things to note about this script:
+
+ 1. Notice that the second query has a `.offset()` function call, this tells the script how far back in time to query.
+ 2. The shift clause then adds an hour to all the timestamps from the second query. This is necessary in order to join the two queries, since Kapacitor can only join on timestamp.
+ 3. Once the data in the first and second query are equivelent, we can join them together and assign them synonyms 'cur' and 'prev'.
```

Comment:
Add code tags to 'cur' and 'prev' to denote that it's code, in addition to the single-quotes which denote that they're names